src/main/java/picard/vcf/CallingMetricAccumulator.java

```diff
@@ -120,6 +120,12 @@ public void accumulate(final VariantContext vc) {
             final String singletonSample = getSingletonSample(vc);
             updateSummaryMetric(summaryMetric, null, vc, singletonSample != null); // The summary metric has no genotype.
 
+            //Use sampleMetricsMap.get on HomRef samples in case that sample isn't ever put in the map
+            vc.getSampleNames().stream()
+                    .filter(sampleName -> vc.getGenotype(sampleName).isHomRef())
```

Comment:
This will add a lot of compute in large call-sets where the vast majority of genotypes are HomRef.

why not change `setup` function to be:

```java
public void setup(final VCFHeader vcfHeader) {
    vcfHeader.getGenotypeSamples().stream().forEach(sampleName -> sampleMetricsMap.get(sampleName));
}
```

and thus avoid calling `get` a lot more times than you intended?